mobility_pipeline/lib/overlap.py

```diff
@@ -0,0 +1,9 @@
+from shapely.geometry import Polygon
+
+def computeOverlap(enclosing, polygons):
```

Comment:
In Python, we generally use `snake_case` instead of `camelCase` for variables and functions. Note that you'll need to make these changes both here in this file and in the tests file. Some of the variables in the function need the same change. Here's what this function declaration would look like with `snake_case`.
```python
def compute_overlap(enclosing, polygons):
```